src/main/java/org/folio/edge/sip2/handlers/InvalidMessageHandler.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.edge.sip2.handlers;
+
+public class InvalidMessageHandler implements ISip2RequestHandler {
+  @Override
+  public String execute(Object message) {
+    return "96\r";
```

Comment:
We will probably want to move the message terminator to being added elsewhere. We need to add a checksum to this message and there should just be a utility method to do this for all messages. Of course, this message cannot be sent without error detection enabled and the only value for the message is "96", so the checksum will be constant. I still think there is benefit to using a common method for this. We don't need to implement it for this PR, but we are getting close (probably when Login is implemented).

Reply:
Agreed. There should be a template that takes care of adding termination delimiter and error detection data given the binding data's state

Reply:
A TODO item for all templates/commands, let's punt it for now.